client/linuxabi/linux_abi.go

```diff
@@ -203,7 +116,7 @@ type TdxQuoteHdr struct {
 	/* Length of Quote, filled by VMM */
 	OutLen uint32
 	/* Actual Quote data or TDREPORT on input */
-	Data [TdQuoteBufSize]byte
+	Data [ReqBufSize]byte
```

Comment:
Can you please confirm Data can accomodate the input report as well as the output quote?

Reply:
Intel [GHCI spec](https://cdrdv2-public.intel.com/726792/TDX%20Guest-Hypervisor%20Communication%20Interface_1.5_348552_004%20-%2020230317.pdf) states about this: 
```
Shared GPA as input – the memory contains a TDREPORT_STRUCT.
The same buffer is used as output – the memory contains a TD Quote.
```

Reply:
Done.